ksql-rest-app/src/main/java/io/confluent/ksql/rest/entity/StreamedRow.java

```diff
@@ -59,6 +60,7 @@ public StreamedRow(
     this.row = row;
     this.errorMessage = errorMessage;
     this.finalMessage = finalMessage;
+    ImmutableList l;
```

Comment:
What is this?

Reply:
Good catch. I think I was just playing with ImmutableList. I'll get rid of it.